pages/networks/network1/resources.md

```diff
@@ -30,19 +29,33 @@ The above info can be found in this [`networks` repository](https://github.com/d
 | WS    | `wss://indexer.dydx.trade/v4/ws` |
 
 
+## StateSync nodes
+| Team           |  URI                                                                                  |
```

Comment:
Please change URI to **Statesync Peers**
Please add a column for region (this may be important to some operators)